packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -0,0 +1,24 @@
+import { useFeature } from '../components/GrowthBookProvider';
+import { feature } from '../lib/featureManagement';
+import { FeedLayout } from '../lib/featureValues';
+
+interface UseFeedLayoutProps {
+  feedName: string;
+}
+
+// TODO: return type to be changed once more info is added to the return object
+export const useFeedLayout = ({
```

Comment:
I'm not 100% sure why we need the hook again might just miss the context or forgot about it.
In either case the naming is a little bit confusing, could you maybe help me understand why we can't simply use the feature as the evaluation?